corehq/apps/app_manager/static/app_manager/js/modules/report_module.js

```diff
@@ -363,7 +363,7 @@ hqDefine('app_manager/js/modules/report_module', function () {
         self.currentModuleFilter.subscribe(self.changeSaveButton);
         $(options.containerId + ' input').on('textchange', self.changeSaveButton);
 
-        function newReport(options) {
+        var newReport = function (options) {
```

Comment:
We don't have an official preference for `function foo() { ... };` versus `var foo = function () { ... };`, either is fine.

Reply:
:+1: